client/session_container.go

```diff
@@ -0,0 +1,59 @@
+package client
+
+import (
+	v2session "github.com/nspcc-dev/neofs-api-go/v2/session"
+	"github.com/nspcc-dev/neofs-sdk-go/session"
+)
+
+// sessionContainer is a special type which unifies session logic management for client parameters.
+// All methods make public, because sessionContainer is included in Prm* structs.
+type sessionContainer struct {
+	isSessionIgnored bool
+	meta             v2session.RequestMetaHeader
+}
+
+// GetSession returns session object.
+//
+// Returns:
+//   - [ErrNoSession] err if session wasn't set.
+//   - [ErrNoSession] or [ErrNoSessionExplicitly] if IgnoreSession was used.
```

Comment:
```suggestion
//   - [ErrNoSessionExplicitly] if IgnoreSession was used.
```

can we do it like that? cause it seems like a `true` statement. two possible errors make me stuck when i read that

Reply:
It make sense, updated